interface/ttHAnalyzer.h

```diff
@@ -77,10 +77,15 @@ class ttHAnalyzer: public Framework::Analyzer {
         BRANCH(leptons, std::vector<ttH::Lepton>);
         BRANCH(met, std::vector<ttH::Met>);
         BRANCH(jets, std::vector<ttH::Jet>);
+
+
+        //std::vector<ttH::Lepton> selElectrons;
+        //std::vector<ttH::Lepton> selMuons;
+        //std::vector<ttH::Lepton> selleptons;
+
         std::vector<ttH::Dilepton> ll;
         std::vector<ttH::DileptonMet> llmet;
         std::vector<ttH::Dijet> jj;
```

Comment:
It is usually desirable to not have whiteline removed/added without a specific reason.

I am usually personally quite indifferent to that, but there are very picky persons here and there :)

The general idea is that if you don't really need to remove/add a whiteline there, then there is no reason for having it.